Zend/zend_compile.h

```diff
@@ -237,7 +237,7 @@ typedef struct _zend_oparray_context {
 /* or IS_CONSTANT_VISITED_MARK                            |     |     |     */
 #define ZEND_CLASS_CONST_IS_CASE         (1 << 6)  /*     |     |     |  X  */
 /*                                                        |     |     |     */
-/* Class Flags (unused: 15,21,30,31)                      |     |     |     */
+/* Class Flags (unused: 15,21,31)                         |     |     |     */
```

Comment:
Hmm, not sure if we can add a new class flag for BC reasons. @krakjoe, @dstogov, thoughts?

Reply:
I afraid, we cannot do this. This is definitely a binary compatibility break, that can affect third-party software compiled for old PHP versions.

Reply:
This is good for master but not for old versions.